activejdbc/src/test/java/org/javalite/activejdbc/AttributeExclusionValidatorTest.java

```diff
@@ -0,0 +1,59 @@
+package org.javalite.activejdbc;
+
+import org.javalite.activejdbc.test.ActiveJDBCTest;
+import org.javalite.activejdbc.test_models.Student;
+import org.javalite.activejdbc.validation.exclusion.AttributeExclusionValidator;
+import org.junit.Test;
+
+import java.util.Date;
+import java.util.Locale;
+
+public class AttributeExclusionValidatorTest extends ActiveJDBCTest {
+
+    @Test
+    public void testWhenAttributeIsExcluded() {
```

Comment:
Also, we have switched to "should" prefix in all tests a few years ago. Here is one example:
https://github.com/javalite/activejdbc/blob/master/activejdbc/src/test/java/org/javalite/activejdbc/BelongsToParentsTest.java#L25

As you can see, the method name starts with "should" as opposed to "test". The "should" is more expressive that "test" an also indicates that the method name is an expectation. 
Please, see more on this here: http://javalite.io/jspec

This is only recommendation :)